underlay/src/main/proto/criteriaselector/configschema/attribute.proto

```diff
@@ -0,0 +1,13 @@
+syntax = "proto3";
+
+package tanagra.configschema;
+
+option go_package = "github.com/DataBiosphere/tanagra/criteriaselector/configschemapb";
```

Comment:
I did the conversion go since I was more familiar with how to do the necessary processing there than in Java, and this is a one time conversion. I figured there's no point in removing the packages on the off chance it came up again.

Reply:
Maybe worth checking in your conversion script for future reference?

Reply:
No, it's mostly dealing with the existing config JSON that was defined in TypeScript so we won't have to worry about anymore. Plus, it probably makes more sense for scripts we're going to keep around to be written in Java for consistency, which will be a lot simpler now that everything is in a consistent format.